CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.5.0
+
+<!-- generated comparing v5.4.4..master -->
+
+_Mar 7, 2022_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- ♿️ made the `Autocomplete` conform to [ARIA combobox 1.2](https://www.w3.org/TR/wai-aria-1.2/#combobox) (#30601) @EdmundMai
```

Comment:
Minor - it's not combobox 1.2, but ARIA 1.2:

```suggestion
- ♿️ made the `Autocomplete` conform to [ARIA 1.2 combobox](https://www.w3.org/TR/wai-aria-1.2/#combobox) (#30601) @EdmundMai
```

All good otherwise.

Reply:
Thanks!